docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -151,9 +151,9 @@ Contact your Customer Success Manager to increase the cluster size beyond the ma
 
 | Cluster size                                                                        |                                 1x |                                  2x |                               3x |                               4x |
 | :---------------------------------------------------------------------------------- | ---------------------------------: | ----------------------------------: | -------------------------------: | -------------------------------: |
-| Max Throughput **Tasks/day** **\***                                                 |                              4.3 M |                               8.6 M |                           12.9 M |                           17.2 M |
-| Max Throughput **Tasks/second** **\***                                              |                                 50 |                                 100 |                              150 |                              200 |
-| Max Throughput **Process Instances/day** **\*\***                                   |                                3 M |                                 6 M |                              9 M |                             12 M |
+| Max Throughput **Tasks/day** **\***                                                 |                                9 M |                                18 M |                             27 M |                             36 M |
+| Max Throughput **Tasks/second** **\***                                              |                                100 |                                 200 |                              300 |                              400 |
+| Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1.296 k |                          1.728 k |
```

Comment:
```suggestion
| Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1,296 k |                          1,728 k |
```
I think the US uses commas as a numerical separator?

Reply:
We do, but in this case, we would use a decimal. When digit grouping, we use a comma.

Reply:
Thanks for the clarification @akeller, resolving this comment without an edit needed 👍